utils/node_local_test.py

```diff
@@ -1412,6 +1412,22 @@ def test_cache(self):
 
         destroy_container(self.conf, self.pool.id(), container)
 
+    @needs_dfuse
+    def test_truncate(self):
+        """Test file read after truncate"""
+
+        filename = os.path.join(self.dfuse.dir, 'myfile')
+
+        fd = open(filename, 'w')
+        fd.write('hello')
+        fd.close()
+        os.truncate(filename, 1024*1024*4)
+        fd = open(filename, 'r')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations